cpp/src/centrality/betweenness_centrality_impl.cuh

```diff
@@ -319,6 +376,7 @@ void accumulate_vertex_results(
       thrust::make_zip_iterator(distances.begin(), sigmas.begin(), deltas.begin()),
       dst_properties.mutable_view());
 
+    // Accumulate deltas into centralities
```

Comment:
You are scanning the entire vertex range in every loop. This is inefficient. You need to update just the values for the frontier.